### PR TITLE
LOAD CSV support for Google Cloud Storage URIs

### PR DESCRIPTION
**Pull request by rsill-neo4j**

### CODE REVIEW

File: modules/ROOT/pages/clauses/load-csv.adoc [hunk 1]
```diff
@@ -158,6 +158,46 @@ Added 4 nodes, Set 8 properties, Added 4 labels
 |===
 ====
 
+[role=label--enterprise-edition label--not-on-aura label--new-5.21]
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/clauses/load-csv.adoc]: are the labels correct? is it enterprise edition? what about aura?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-980.surge.sh

Updated pages:
- [LOAD CSV](https://neo4j-docs-cypher-980.surge.sh/cypher-manual/5/clauses/load-csv)

File: modules/ROOT/pages/clauses/load-csv.adoc (hunk 1 above)
Comment (docrozza) [reply modules/ROOT/pages/clauses/load-csv.adoc]: Correct - enterprise only and not on Aura

File: modules/ROOT/pages/clauses/load-csv.adoc [hunk 2]
```diff
@@ -158,6 +158,46 @@ Added 4 nodes, Set 8 properties, Added 4 labels
...
+1,ABBA,1992
+2,Roxette,1986
+3,Europe,1979
+4,The Cardigans,1992
+----
+
+.Query
+[source, cypher, role=test-skip]
+----
+LOAD CSV FROM 'gs://artists.csv' AS row
```
Comment (docrozza) [inline modules/ROOT/pages/clauses/load-csv.adoc]: URI needs to be in the form 'gs://somebucketname/artists.csv' as the first segment of the URI path identifies the bucket to use

Noticed that the S3 docs below have a similar problem, i.e. they also need a bucket name like  's3://somebucketname/artists.csv'

Comment (rsill-neo4j) [reply modules/ROOT/pages/clauses/load-csv.adoc]: while at it, since this is also cloud storage, do you think we can conflate the two sections?

Comment (docrozza) [reply modules/ROOT/pages/clauses/load-csv.adoc]: Might be nice to - though not sure the best way to write it though 😄 

Maybe have `gs://eighties/artists.csv` with some data and `s3://nineties/artists.csv` with some different data and have the LOAD CSV parameterised with those URIs?

```
UNWIND ['gs://eighties/artists.csv', 's3://nineties/artists.csv'] AS cloudUri
LOAD CSV FROM cloudUri AS row
```

Comment (rsill-neo4j) [reply modules/ROOT/pages/clauses/load-csv.adoc]: i added a section heading instead
the UNWIND query looks cool though :)

File: modules/ROOT/pages/clauses/load-csv.adoc [hunk 3]
```diff
@@ -158,8 +158,50 @@ Added 4 nodes, Set 8 properties, Added 4 labels
...
```
Comment (docrozza) [inline modules/ROOT/pages/clauses/load-csv.adoc]: gs://gs-bucket/artists.csv

File: modules/ROOT/pages/clauses/load-csv.adoc [hunk 4]
```diff
@@ -158,8 +158,50 @@ Added 4 nodes, Set 8 properties, Added 4 labels
...
+2+d|4 rows
+
+Added 4 nodes, Set 8 properties, Added 4 labels
+|===
+====
+
+
 [role=label--enterprise-edition label--not-on-aura label--new-5.19]
-=== Import from an AWS S3 URI
+==== Import from an AWS S3 URI
```
Comment (docrozza) [inline modules/ROOT/pages/clauses/load-csv.adoc:205]: also s3://aws-bucket/artists.csv in the header name

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-980.surge.sh

Updated pages:
- [LOAD CSV](https://neo4j-docs-cypher-980.surge.sh/cypher-manual/5/clauses/load-csv)

File: modules/ROOT/pages/clauses/load-csv.adoc [hunk 5]
```diff
@@ -158,15 +158,57 @@ Added 4 nodes, Set 8 properties, Added 4 labels
 |===
 ====
 
+=== Import from cloud URIs
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/load-csv.adoc:162]:
```suggestion
[role=label--enterprise-edition label--not-on-aura]
=== Import from cloud URIs
```

File: modules/ROOT/pages/clauses/load-csv.adoc [hunk 6]
```diff
@@ -158,15 +158,57 @@ Added 4 nodes, Set 8 properties, Added 4 labels
...
+
+[role=label--enterprise-edition label--not-on-aura label--new-5.21]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/load-csv.adoc]:
```suggestion
[role=label--new-5.21]
```

File: modules/ROOT/pages/clauses/load-csv.adoc [hunk 7]
```diff
@@ -158,15 +158,57 @@ Added 4 nodes, Set 8 properties, Added 4 labels
...
+| 'Europe' | '1979'
+| 'The Cardigans' | '1992'
+2+d|4 rows
+
+Added 4 nodes, Set 8 properties, Added 4 labels
+|===
+====
+
+
 [role=label--enterprise-edition label--not-on-aura label--new-5.19]
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/clauses/load-csv.adoc]:
```suggestion
[role=label--new-5.19]
```

**Review by JPryce-Aklundh — COMMENTED** [review]
Looks good :) a few suggestions from about labels. maybe put Google Cloud storage after AWS - both introduced in a later version and from alphabetical order pov.
Also, please add an entry to the Additions page under 5.21 -> New Features.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-980.surge.sh

Updated pages:
- [LOAD CSV](https://neo4j-docs-cypher-980.surge.sh/cypher-manual/5/clauses/load-csv)

**Review by docrozza — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-980.surge.sh

Updated pages:
- [LOAD CSV](https://neo4j-docs-cypher-980.surge.sh/cypher-manual/5/clauses/load-csv)
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-980.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by JPryce-Aklundh — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**